kubernetes/samples/scripts/common/utility.sh

```diff
@@ -333,8 +333,11 @@ function createFiles {
     cp ${domainPropertiesInput} ${domainPropertiesOutput}
     sed -i -e "s:%DOMAIN_NAME%:${domainName}:g" ${domainPropertiesOutput}
     sed -i -e "s:%ADMIN_PORT%:${adminPort}:g" ${domainPropertiesOutput}
+    sed -i -e "s:%ADMIN_SSL_PORT%:${adminSslPort}:g" ${domainPropertiesOutput}
     sed -i -e "s:%ADMIN_SERVER_NAME%:${adminServerName}:g" ${domainPropertiesOutput}
     sed -i -e "s:%MANAGED_SERVER_PORT%:${managedServerPort}:g" ${domainPropertiesOutput}
+    sed -i -e "s:%MANAGED_SERVER_SSL_PORT%:${managedServerSslPort}:g" ${domainPropertiesOutput}
+    sed -i -e "s:%USE_KSS_FOR_DEMO%:${useKSSForDemo}:g" ${domainPropertiesOutput}
```

Comment:
I'd like to understand why this is a "DEMO".  I'm fine with having sample credentials as we do for SSL demo certificates.  But, it's not clear why that would be the case here.